### PR TITLE
Week 9 Simon and Jennys clone of spotify

### PR DESCRIPTION
**Pull request by Kyparn**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/public/icons/spotify.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="800" width="1200" viewBox="-33.4974 -55.829 290.3108 334.974"><path d="M177.707 98.987c-35.992-21.375-95.36-23.34-129.719-12.912-5.519 1.674-11.353-1.44-13.024-6.958-1.672-5.521 1.439-11.352 6.96-13.029 39.443-11.972 105.008-9.66 146.443 14.936 4.964 2.947 6.59 9.356 3.649 14.31-2.944 4.963-9.359 6.6-14.31 3.653m-1.178 31.658c-2.525 4.098-7.883 5.383-11.975 2.867-30.005-18.444-75.762-23.788-111.262-13.012-4.603 1.39-9.466-1.204-10.864-5.8a8.717 8.717 0 015.805-10.856c40.553-12.307 90.968-6.347 125.432 14.833 4.092 2.52 5.38 7.88 2.864 11.968m-13.663 30.404a6.954 6.954 0 01-9.569 2.316c-26.22-16.025-59.223-19.644-98.09-10.766a6.955 6.955 0 01-8.331-5.232 6.95 6.95 0 015.233-8.334c42.533-9.722 79.017-5.538 108.448 12.446a6.96 6.96 0 012.31 9.57M111.656 0C49.992 0 0 49.99 0 111.656c0 61.672 49.992 111.66 111.657 111.66 61.668 0 111.659-49.988 111.659-111.66C223.316 49.991 173.326 0 111.657 0" fill="#1ed660"/></svg>
```
Comment (Neaa99) [inline code/public/icons/spotify.svg:1]: Nice with changing the size and color inside of the svg file!

File: code/src/App.js [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
+            img={album.images[1].url}
+            title={album.name}
+            albumLink={album.external_urls.spotify}
+            artists={album.artists}
+            key={album.name}
+          />
+        ))}
+      </div>
+      <Footer />
+    </main>
```
Comment (Neaa99) [inline code/src/App.js:28]: Cooll that you changed the div to main. Good replacement! Also very clean code in the App file

File: code/src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+// This is the footer part with links to our githubs acconts
+
+const Footer = (props) => {
```
Comment (Neaa99) [inline code/src/components/Footer.js:5]: Nice with adding a footer!

File: code/src/components/Icons.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+// here we have the icons that are ontop of the artist card with hover effect
+const Icons = (props) => {
+  return (
+    <div className="icons">
+      <img
+        className="heart-icon"
+        src="/icons/heart.svg"
+        alt="images of heart"
+      />
+      <img className="play-icon" src="/icons/play.svg" alt="images of heart" />
```
Comment (Neaa99) [inline code/src/components/Icons.js:11]: It would be good to do a a tag here, and link the play button the the song url

File: code/src/index.css [hunk 5]
```diff
@@ -1,13 +1,221 @@
...
+  width: 100px;
+}
+
+/* ----- ICONS -----*/
+
+.icons {
+  width: 40px;
+  z-index: 10;
+  position: absolute;
+  opacity: 0.5;
```
Comment (Neaa99) [inline code/src/index.css:65]: Why opacity 0.5 here? I noticed that the icons were a bit faded. Maybe chenge this to opacity 1?

File: code/src/index.css [hunk 6]
```diff
@@ -1,13 +1,221 @@
...
+}
+
+.singer:nth-last-of-type(2)::after {
+  content: ' & ';
+}
+
+.singer:first-child::before {
+  content: ' ';
+}
+
```
Comment (Neaa99) [inline code/src/index.css:152]: Great that you was able to do this in CSS insted of js!

File: code/src/index.css [hunk 7]
```diff
@@ -1,13 +1,221 @@
...
+  opacity: 0.5;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  top: 40%;
+  left: 45%;
+  opacity: 0;
+  grid-gap: 30px;
+  gap: 30px;
+}
```
Comment (Neaa99) [inline code/src/index.css:170]: Here you repeat the .icons. Is there a specific reason why? Maybe merge rhis part with the .icons part higher up so you don't repeat youself

**Review by Neaa99 — COMMENTED** [review]
Hi Jenny and Simon! Very nice clean site. You reached the goals for this project and you have copied the design pretty well. If I would give you some advice is that lower the font-size on the header (since it's smaller on the design) and take a look at the opacity on the icons, since it's a bit faded and hard to se. Other than that, the site works great and the code looks good besides from a few repeats. Great work!!

File: code/src/index.css (hunk 5 above)
Comment (JustineZwiazek) [reply code/src/index.css:65]: Good pointer here from Linnea, I agree!

File: code/src/index.css (hunk 6 above)
Comment (JustineZwiazek) [reply code/src/index.css:152]: Yay! we took a similar approach on our project as well.

**Comment by JustineZwiazek** [comment]
Really great job Jenny and Simon! I think your project satisfies the requirements, and I think that the hover picture opacity is on point! One pointer I wanted to add on the design side(sorry, I didn't have the option to comment it in in the right place, doing the review as the 2nd person) would be that spacing between the song "tiles" could be a bit smaller. I think that would make it look even closer to the Techigo design. Well done! I am so proud of how great our team is doing :)

**Pull request closed without merging**